src/utils/error.ts

```diff
@@ -13,28 +9,6 @@ export const prettifyError = (error: unknown): string => {
   return stringify(error);
 };
 
-export const prettifyTransactionError = async (
```

Comment:
This function was used to prettify an error. But the "simulate" step below wasn't used since v5 isn't returning ethers error codes anymore. Moving the insufficient funds logic inline in SendTransactionWorker.